packages/ui/src/views/repo/repo-branch-rules/components/repo-branch-rules-data.tsx

```diff
@@ -0,0 +1,64 @@
+import { MergeStrategy } from '@views/repo/pull-request'
+
+import { BranchRuleId } from '../types'
+
+export const branchRules = [
```

Comment:
any type needed on this?

Reply:
TS seems happy for now, lets add one in a follow up PR if its not a blocker for this one.